pycbc/waveform/parameters.py

```diff
@@ -417,32 +417,34 @@ def docstr(self, prefix='', include_label=True):
                 dtype=float, default=0., label=r"$\delta$",
                 description="Mean anomaly of the periastron (rad).")
 tc = Parameter("tc",
-                dtype=float, default=None, label=r"$t_c$ (s)",
-                description="Coalescence time (s).")
+               dtype=float, default=None, label=r"$t_c$ (s)",
+               description="Coalescence time (s) in "
+                           "geocentric/SSB/LISA coords.")
```

Comment:
@WuShichao Looking at this, I'd take out reference here to coordinates systems generally. I think it's going to end up being confusing.